tools/install-dependencies

```diff
@@ -68,6 +68,7 @@ tar xzf protobuf-java-$PROTOBUF_VERSION.tar.gz
 # Build Protobuf
 cd protobuf-$PROTOBUF_VERSION
 ./configure --prefix="$PREFIX"
+make clean
```

Comment:
maybe before `./configure`?

Reply:
it does seem to work too but the ci for android and ios fails with `make: *** No rule to make target `clean'.  Stop.`